core/trigFilters.fcl

```diff
@@ -1007,10 +1342,10 @@ TrigFilters : {
    filters : {
       @table::TrigCprFilters
       @table::TrigTprFilters
+      @table::TrigAprFilters
```

Comment:
Does this mean the trigger outputs will include APR?  Our existing Sim Production scripts select outputs based on trigger paths matching *_HighP_StopTarg.  We don't want to change this for the redigitization campaign.
Can the APR triggers be put into a 'Development' or 'Test' menu?  That would allow comparing them with the existing menu without overwriting it.

Reply:
I thought we weren't splitting anymore the MC data based on the trigger decision. Is this not the case?